CONTRIBUTING.md

```diff
@@ -91,6 +74,6 @@ Please contribute! The Layer5 site uses GitHub Pages (Jekyll) to host its site.
 `make site` # this will run a local web server with "live reload" conveniently enabled.
 1. Commit and push changes to your remote branch.
 `git push origin <my-username>/<my-changes>`
-1. Open a [pull request](../../pulls) (in your web browser) against the master branch on https://github.com/layer5io/layer5.
+1. Open a [pull request](../../pulls) (in your web browser) against the `layer5-ng` branch on https://github.com/layer5io/layer5.
 
-Build and preview your changes by running `make site` or `make docker`.
```

Comment:
Are we able to run the new site in docker?

Reply:
No, not yet..

Reply:
Oh, I see. Ok.